packages/ai-bot/lib/matrix.ts

```diff
@@ -132,6 +122,54 @@ export async function sendError(
   }
 }
 
+export const toMatrixMessageContent = (
+  functionCall: FunctionToolCall,
+  eventToUpdate: string | undefined,
+): IContent | undefined => {
+  let { arguments: payload } = functionCall;
+  if (functionCall.name === 'patchCard') {
+    const id = payload['card_id'];
+    const body = payload['description'] || "Here's the change:";
```

Comment:
this is an example of why the `agruments` type is not `any`. you have very specific assumptions around properties in the arguments when the type if `patchCard`--please assert these into the type for `arguments`

Reply:
As in the previous PR, We will keep all this code in the command service for now until the command field api is introduced. 

payload is the name also referring now to the arguments returned by the ai bot